install/linux/usr/share/odemis/sim/enzel-sim.odm.yaml

```diff
@@ -354,6 +372,7 @@ ENZEL-Sim: {
     # It should be within the ACTIVE_RANGE
     FAV_POS_ACTIVE: { 'rx': 0.4886, 'rz': 0, 'x': 0.0, 'y': 0.00, 'z': 0.002 },
     # SEM Imaging position
+    # Unused
```

Comment:
If it is unused should there be a todo which states that this key can be removed in the future?